.travis.yml

```diff
@@ -1,3 +1,4 @@
 language: php
 php:
-  - 7.1
+  - 7.4
```

Comment:
We require php ^7.3 in composer.json so it seems like we should check 7.3 7.4 and 8.0 (and 8.1?)

Reply:
Fixed! f51e95c5a2f45d8c5ccd1dd02f1433ede8e76200